src/NLog.Extensions.Logging/LogEventInfoExtensions.cs

```diff
@@ -0,0 +1,36 @@
+using Microsoft.Extensions.Logging;
+
+namespace NLog.Extensions.Logging
+{
+
+    public static class LogEventInfoExtensions {
+
+
+        /// <summary>
+        /// Gets the <see cref="Microsoft.Extensions.Logging.EventId"/>. If it was not included in the log 
+        /// event, then uses the default value.
+        /// </summary>
+        /// <param name="logEvent"></param>
+        /// <param name="defaultValue"></param>
+        /// <returns></returns>
+        public static EventId getEventId(this LogEventInfo logEvent, EventId defaultValue = default(EventId))
+        {
+
+            if (!logEvent.HasProperties)
+                return defaultValue;
+
+            // setup key names the same way "NLogLogger.cs" does it
+            var options = new NLogProviderOptions();
+            var idKey = $"EventId{options.EventIdSeparator}Id";
+            var nameKey = $"EventId{options.EventIdSeparator}Name";
```

Comment:
no need for these 2 lines. Just use `EventId` key as that one holds the object. See https://github.com/NLog/NLog.Extensions.Logging/blob/014577574e62134e5d141502d14fedea90819f89/src/NLog.Extensions.Logging/NLogLogger.cs#L39